File: src java/Sum.java
public class Sum { //2+4+8+16....+256
    public static void main(String[] args) {
        int a = 1;
        int sum=a;
        while (a < 256) {
            a = a * 2;
            sum=sum+a;
            System.out.println(a+" - "+sum);

        }
        System.out.println("-----------");
        System.out.println("Итог: " + sum);
    }
}

File: src/Sum.kt
//2+4+8+16....+256
class Sum {
    fun sumKotlin() {
        var a = 1
        var sum = a
        while (a < 256) {
            a *= 2
            sum += a
            println("$a - $sum")
        }
        println("-----------")
        println("Итог: $sum")
    }
}